alacritty/src/logging.rs

```diff
@@ -108,7 +108,29 @@ impl log::Log for Logger {
         }
 
         let now = time::strftime("%F %T.%f", &time::now()).unwrap();
-        let msg = format!("[{}] [{:<5}] [{}] {}\n", now, record.level(), target, record.args());
+
+        let lines = record.args().to_string();
+        let mut lines = lines.split('\n').peekable();
+
+        let mut msg = format!("[{}] [{:<5}] [{}] ", now, record.level(), target);
+
+        // Alignment for the lines after the first new line character in the payload.
+        let alignment = msg.chars().count();
```

Comment:
Just to be clear, this should be fine since we don't have to deal with fullwidth/unicode here. In fact I think `msg.len()` should probably work too and be a little easier? We're only dealing with ascii here. Technically `msg.chars().count()` is slightly more robust, but if we want to be able to handle non-unicode stuff we'd need to consider fullwidth characters?